packages/@okta/vuepress-site/docs/guides/device-context/main/index.md

```diff
@@ -0,0 +1,151 @@
+
+Administrators can write authentication policy rules for sign-in requests from a new device using the **new device behavior**. The following diagram shows this flow:
+
+<div class="full">
+
+![Flow diagram showing how the new device behavior works](/img/advanced-use-cases/device-context-new-behavior-flow.png)
+
+</div>
+
+> **Note:** Find your Org's definition of **New Device** in your Admin Console under **Security** > **Behavior Detection**.
+
+## Update your application to send an X-Device-Token header
+
+<StackSnippet snippet="integrationsteps" />
+
+## An Example: Adaptive Multi-Factor Authentication
+
+An Adaptive Multi-Factor Authentication (AMFA) policy means that users are prompted for a different number of authentication factors based on a number of criteria, including:
+
+* Location: Where are they signing in from?
+* New device: Are they signing in from a managed or known device?
+* Network: Are they signing in from a safe network?
+* Travel: Are they signing in from a new location?
+* What groups does the user belong to?
+* Are they signing in through SSO?
+
+For example, a user signing in to your website from a new device or new country may signal a risky situation. Requiring additional authenticators&mdash;email, phone SMS, or biometrics&mdash;gives extra identity assurance in these circumstances.
+
+In this example, you'll implement a simple AMFA policy requiring two authentication factors from new devices and only one factor from known devices. The following diagram illustrates this flow:
+
+<div class="full">
+
+![Flow diagram showing how adaptive multifactor authentication works for new devices](/img/advanced-use-cases/device-context-adaptive-mfa-flow.png)
+
+</div>
+
+Follow these steps to enable this AMFA policy in your app:
+
+1. Update your application to send the `X-Device-Token` header to your Org.
+2. Update or create a new global session policy to require a second authentication factor for sign-in requests from new devices.
+3. Update or create an authentication policy for your app to require and identify a second authentication factor for new devices.
+
+> **Note**: For testing purposes, we suggest creating new policies rather than adding AMFA rules to your existing policies.
+
+### Create a new global session policy for AMFA
+
+1. Open the **Admin Console** for your org.
+2. Choose **Security** > **Global Session Policy** to show the available global session policies.
+3. Click **Add Policy**.
+4. Set a **Policy Name**; for example, "New Device AMFA Policy".
+5. Set **Assign to Groups** to your target user groups; for example, "Test Users".
+6. Click **Create Policy and Add Rule** to save the new policy.
+
+After creating the policy, the **Add Rule** dialog appears.
+
+1. Set a **Rule Name**; for example, "New Device AMFA Rule".
+2. Set **Behavior is** to **New Device**. This is where the `X-Device-Token` header is checked.
```

Comment:
I don't see the "Behavior is" field in my org. Probably because I don't have the feature enabled but just wanted to let you know so you can confirm.

Reply:
Here it is in my org

![image](https://user-images.githubusercontent.com/97525701/189139899-6d5abce9-33d3-4dfe-a7e9-24a4b174e779.png)